codespeed/mercurial.py

```diff
@@ -45,8 +45,7 @@ def getlogs(endrev, startrev):
     updaterepo(endrev.branch.project, update=False)
 
     cmd = ["hg", "log",
-            "-r", "%s:%s" % (endrev.commitid, startrev.commitid),
-            "-b", "default",
```

Comment:
If the default branch is called something else than "default" it will now work. But if we configure and save data on codespeed from another branch it will still break because the actual mercurial repo is always on the default branch. Or am I missing something here?